packages/types/src/constants.ts

```diff
@@ -20,3 +20,7 @@ export const MIGRATION_STEPS = ['renameAuto', 'renameManual', 'renameSupport', '
 export type MigrationStep = typeof MIGRATION_STEPS[number]
 
 export const PACKAGE_MANAGERS = ['npm', 'yarn', 'pnpm'] as const
+
+// Note: ONLY change this in code that will be merged into a release branch
+// for a new major version of Cypress
+export const MAJOR_VERSION_FOR_CONTENT = '11'
```

Comment:
Does it make sense to add a reference to the locales file where there are also hard coded version numbers?

Reply:
Good catch, that's not used any more - it's hanging around from before I moved it to this file. Removed.